hystrix-core/src/main/java/com/netflix/hystrix/strategy/HystrixPlugins.java

```diff
@@ -295,6 +345,32 @@ private static Object getPluginImplementationViaArchaius(Class<?> pluginClass) {
             return null;
         }
     }
+    
+    
+
+    private static HystrixDynamicProperties resolveDynamicProperties(ClassLoader classLoader) {
```

Comment:
See my other line comment on logging. We have to be very careful about when we kick off logging. It needs to be after whatever configuration framework has loaded (`HystrixDynamicProperties`) because the configuration framework may want to participate in configuring logging. This is a long going chicken/egg problem with java config and logging. This appears to be after so we can probably do it **but we cannot statically call `getLogger` or else we load the logging framework too early.**  I'll see if I can put in a safe change.